autoshop/api/v1/serializers.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import serializers
+
+from core.models import Profile
+
+
+class UserProfileSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Profile
+        fields = '__all__'
```

Comment:
we still need to figure out how to include fields `first_name` and `last_name` with this API